docs/kyma/docs/017-details-local-installation.md

```diff
@@ -0,0 +1,68 @@
+---
+title: Local installation
+type: Details
+---
+
+For better understanding of complex installation process, this guide describes it step by step.
+
+## Start local installation
+
+To fire local installation run following command:
+```
+./installation/cmd/run.sh
+```
+
+This script sets up default parameters, starts Minikube, builds Kyma-Installer, generates local configuration, creates `installer` Custom Resource and sets up Installer. Subsequent sections provide a detailed description of each step.
+
+### Installation parameters
+
+You can execute the `installation/cmd/run.sh` script with the following parameters:
+
+- `--skip-minikube-start` - it skips execution of the `installation/scripts/minikube.sh` script. See the "Start Minikube" section for more details.
+- `--vm-driver` -  either `virtualbox` or `hiperkit` depending on your operating system
+
+## Start Minikube
+
+> **NOTE:** To work with Kyma, use only the provided scripts and commands. Kyma does not work on a basic Minikube cluster that you can start using the `minikube start` command or stop with the `minikube stop` command. If you don't need Kyma on Minikube anymore, remove the cluster with the `minikube delete` command.
+
+Purpose of `installation/scripts/minikube.sh` is to configure and start Minikube. The script also checks if your development environment is cofigured to handle Kyma installation. This includes checking Minikube and Kubectl versions. In case Minikube is already initialized you will be prompted to agree to remove previous Minikube cluster. Script exits if you don't want to restart your cluster.
+
+>**NOTE:** Minikube is configured to disable default nginx ingress controller.
+
+>**NOTE:** For the complete list of parameters passed to `minikube start` command, refer to the `installation/scripts/minikube.sh` script.
+
+Once Minikube is up and running, the script adds local installation entries to /etc/hosts.
+
+## Build Kyma-Installer
+
+Installer is application based on a [Kubernetes operator](https://coreos.com/operators/). Its purpose is to install Helm charts defined in the Installer Custom Resource. Kyma-Installer is a Docker image that bundles Installer binary with Kyma charts. 
```

Comment:
> Installer is an application based on a [Kubernetes Operator](https://coreos.com/operators/). Its purpose is to install Helm charts defined in the `Installation` Custom Resource. The Kyma installer is a Docker image that bundles the Installer binary with Kyma charts. 

- the `Installation` Custom Resource, right? If anything else, pls adjust it in the whole text to be consistent.
- the Kyma installer, right?  If anything else, pls adjust it in the whole text to be consistent.